rudderanalytics-client/src/main/java/com/rudderstack/sdk/java/FlushBlocking.java

```diff
@@ -0,0 +1,54 @@
+package com.rudderstack.sdk.java;
+import java.util.concurrent.Phaser;
+import com.rudderstack.sdk.java.RudderAnalytics;
+import com.rudderstack.sdk.java.Callback;
+import com.rudderstack.sdk.java.MessageTransformer;
+import com.rudderstack.sdk.java.Plugin;
+import com.rudderstack.sdk.java.messages.MessageBuilder;
+import com.rudderstack.sdk.java.messages.Message;
+
+public class FlushBlocking {
+
+	  public static FlushBlocking create() {
+	    return new FlushBlocking();
+	  }
+
+	  FlushBlocking() {
+	    this.phaser = new Phaser(1);
+	  }
+
+	  final Phaser phaser;
+
+	  public Plugin plugin() {
```

Comment:
As this will be used internally in the SDK, we don't need `public` here. It can be package private.

Reply:
updated